modules/inputSpinner/InputSpinner.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import {
+  ThemeStyleSheetFactory,
+  StandardProps,
+  WithStylesProps
+} from "@diana-ui/types";
+import { withStyles } from "@diana-ui/base";
+import { ButtonText } from "@diana-ui/typography";
+
+// @ts-ignore
```

Comment:
do we need this ?

Reply:
To be able to use onChange with different type. Thats a problem with typescript